### DISCUSSION TITLE
The win 10 taskbar should show current layout

### DISCUSSION THREAD

**Discussion by Feliks-WR** [valinet/ExplorerPatcher#3250] [Ideas]
The win 11 one does, and I use multiple keyboard layouts, like many other *devs* (dvorak, qwerty, programmer dvorak).
If win 10 taskbar showed layout, it would be ***the ultimate thing***!

**Comment by pyrates999** [comment]
what do you mean?

**Comment by Feliks-WR** [comment]
[cid:1098db92-ea8b-4f66-8955-951f73b69e88]
________________________________
منجانب: pyrates999 ***@***.***>
‏‏بھیجا گیا: 08 مئی, 2024 16:37
بطرف: valinet/ExplorerPatcher ***@***.***>
Cc: Feliks-WR ***@***.***>; Author ***@***.***>
‏‏موضوع: Re: [valinet/ExplorerPatcher] The win 10 taskbar should show current layout (Discussion #3250)


what do you mean?

**Comment by pyrates999** [comment]
You can't upload images to here by replying via email.

**Comment by Feliks-WR** [comment]
![thumbnail_image](https://github.com/valinet/ExplorerPatcher/assets/73439021/a84263be-3bf8-4c59-a718-2b4a4e741685)

**Comment by pyrates999** [comment]
According to this: https://github.com/valinet/ExplorerPatcher/issues/3009

It's been fixed in the latest release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.2_c245b8f

If that doesn't fix it, try installing the latest pre-release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.3_d398195

Then if that doesn't fix it:

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.
